scc/light_client/light_client.go

```diff
@@ -21,6 +22,9 @@ type LightClient struct {
 type Config struct {
 	Url     *url.URL
 	Genesis scc.Committee
+	// unless specified only try once without delays
```

Comment:
nit: since technically these fields are always set to some value (=specified), it would be better to state how the zero value is interpreted.

Reply:
https://github.com/0xsoniclabs/sonic/pull/128/commits/91dfbe1c5d1a763e2d73c147f8e929977c9b9f00